client/src/index.js

```diff
@@ -1,6 +1,12 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import * as Sentry from "@sentry/browser";
+
 import "./index.css";
 import App from "./App";
 
+Sentry.init({
```

Comment:
i think this should be called from inside the `componentDidCatch` lifecycle in the App component, so you can catch the error